modules/core/src/main/scala/sangria/package.scala

```diff
@@ -0,0 +1,15 @@
+/** Sangria GraphQL server library.
+  *
+  * Sangria is a library that provides parsing, validation, execution and other services for GraphQL
+  * queries.
+  *
+  * It typically requires other libraries to build a complete GraphQL service—including, perhaps,
+  * ones that provide a service interface (e.g.,
+  * [[https://doc.akka.io/docs/akka-http/current/index.html Akka HTTP]]) and a database interface
+  * (e.g., [[https://scala-slick.org/ Slick]])—as well as custom code to bridge the gap between data
+  * representations that are natural to the database vs. to the GraphQL schema.
```

Comment:
My experience is that new users read the website and not the scaladoc.
So I'd prefer not linking to any other libraries here, keeping the sources neutral.

```suggestion
  * It typically requires other libraries to build a complete GraphQL service
  * —including, perhaps, ones that provide a HTTP service interface and a database interface—
  * as well as custom code to bridge the gap between data representations that are natural
  * to the database vs. to the GraphQL schema.
```

Reply:
I also recognize that new users use the website docs first. I don't see how that's relevant. Yes, this scaladoc is for the users who are digging for a deeper understanding.

I felt that pointing to a specific example of "an HTTP service interface" was clearer than using a term ("HTTP service interface") that may be ambiguous to the reader. I picked the ones that I thought most users would know. My text first read like yours, without the "(e.g., …)"s.

Reply:
You can change the text how you like.
I'd just like that we don't mention any other libraries / projects to stay neutral.

Reply:
Can you please change that?

Reply:
> You can change the text how you like.

I thought you were giving me the choice.

Reply:
You can change the text how you like *as long as* we don't mention any other libraries / projects to stay neutral. 😉